pkg/sqlite/load.go

```diff
@@ -424,7 +426,7 @@ func (s *sqlLoader) AddPackageChannelsFromGraph(graph *registry.Package) error {
 	// update each channel's graph
 	for channelName, channel := range graph.Channels {
 		currentNode := channel.Head
-		depth := 1
+		depth := startDepth
```

Comment:
I assume `depth := 1` is a bug and we want the depth starting with 0?

Reply:
Correct.